src/Tizen.Messaging/Tizen.Messaging.Messages/Message.cs

```diff
@@ -131,6 +131,21 @@ private void AddAddress(MessagesAddress address)
             }
         }
 
+        public int GetMessageCount(int mbox, int messageType)
+        {
+            int count;
+            Log.Error(Globals.LogTag, "GetMessageCount is Called");
+            int ret = Interop.Messages.GetMessageCount(_messageHandle, mbox, messageType, out count);
```

Comment:
`Interop.Messages.GetMessageCount` use with message service handle but, you passed a message handle

PLEASE, Test your code before submit.